nexus/variables.tf

```diff
@@ -78,3 +78,15 @@ variable "is_production" {
 variable "nexus_secrets_arn" {
   type = string
 }
+
+variable "is_nexus_openscience_running" {
+  description = "Whether OpenScience nexus should be running"
+  type        = bool
+  default     = true
```

Comment:
I would remove default values, IMO it is better if the plan/apply fails before even starting than an unwanted outcome because of a missing variable in the `<enviroment>.tfvars` file.

Reply:
Good point - that would have revealed issues I had in development a lot earlier too 😭